src/js/reducers.js

```diff
@@ -40,6 +40,18 @@ function newAppState () {
         backSeekIndicator: {type: "TRACK", seekTime: null},
         isDisconnected: false,
         displayLayout:  null,
+        speak: {
+            msgID: null,
+            playTone: false,
+            speakType: "BOTH",
+            ttsChunks: []
+        },
+        resetTimeout: {
+            resetTimeoutValue: 10000
+        },
+        alertTimeoutReseted: {
+            isAlertTimeoutReseted: true
+        },
```

Comment:
@iCollin value of resetTimeoutValue is default value for input box, in this way we can change it and use it in many RPC.

Reply:
The object you added it to is a state that exists per each app. This value should not be different for each app because it has one input and value. 

It should be moved to system state or another single state instance.

Reply:
@iCollin I moved it to system state, please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/22/commits/33db5d59661702666e2a6a6be0167a5442d624fe).